src/tests/unit/onbase-dao.test.js

```diff
@@ -0,0 +1,243 @@
+      stubAxios,
+    ));
+  it('getDocumentKeywords should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.getDocumentKeywords('fake token', 'fake fbLb',
+        'fake', 'dummy id', 'fake keyword collection'),
+      fakeData.data,
+      stubAxios,
+    ));
+  it('getDocumentKeywordTypes should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao2.getDocumentKeywordTypes('fake token', 'fake fbLb',
+        [keyword1, keyword2]),
+      [keyword1, keyword2],
+      stubAxios2,
+    ));
+  it('patchDocumentKeywords should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.patchDocumentKeywords('fake token', 'fake fbLb',
+        'fake',
+        { items: [[keyword1, keyword2]] },
+        [keyword1, keyword2]),
+      fakeData.data,
+      stubAxios,
+    ));
+  it('getDocumentContent should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.getDocumentContent('fake token', 'fake fbLb', 'dummy id'),
+      fakeData,
+      stubAxios,
+    ));
+  it('getDocumentTypeByName should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.getDocumentTypeByName('fake token', 'fake fbLb', docType1.id),
+      docType1.id,
+      stubAxios,
+    ));
+  it('getDocumentTypeByName more than one document type should fail',
+    () => {
+      fakeData.data.items = [docType1, docType1];
+
+      testResultError(
+        onbaseDao.getDocumentTypeByName(
+          'fake token', 'fake fbLb', docType1.id,
+        ),
+        'More than one document types matched.',
+        stubAxios,
+      );
+    });
+  it('getDocumentTypeByName no document type should fail',
+    () => {
+      fakeData.data.items = [];
+
+      testResultError(
+        onbaseDao.getDocumentTypeByName(
+          'fake token', 'fake fbLb', docType1.id,
+        ),
+        'Please provide a valid document type.',
+        stubAxios,
+      );
+    });
+  it('getKeywordTypesByNames should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao2.getKeywordTypesByNames('fake token', 'fake fbLb',
+        {
+          keywordTypeNames: [keywordType2.name],
+          keywordValues: ['0'],
+        }),
+      { KeywordType2: { value: '0', id: keywordType2.id } },
+      stubAxios2,
+    ));
+  it('createQuery should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.createQuery('fake token', 'fake fbLb', 'fake 1', 'fake 1', 'fake 1', 'fake 1'),
+      'fake id',
+      stubAxios,
+    ));
+  it('getQueryResults should return a valid result',
+    () => testResultWithCookie(
+      onbaseDao.getQueryResults('fake token', 'fake fbLb', 'fake id'),
+      fakeData,
+      stubAxios,
+    ));
+  it('getDocumentsByIds should return a valid result',
+    () => testResultData(
+      onbaseDao.getDocumentsByIds('fake token', 'fake fbLb', 'fake id'),
+      stubAxios,
+    ));
+});
```

Comment:
Besides DAO tests, you also need to test the serializers. Also, I updated `getAccessToken` function already to address users who might use the wrong stored cached credential, please make sure your code tested that part and reflect the change.